script.js

```diff
@@ -0,0 +1,79 @@
+const DefaultBooks = [
+  {
+    id: 0,
+    title: 'JavaScript',
+    author: 'Brenden Eich',
+  },
+  {
+    id: 1,
+    title: 'HTML/CSS',
+    author: 'HÃ¥kon Wium Lie',
+  },
+];
+
+// converting json value into object using parse
+let BooksArray = JSON.parse(localStorage.getItem('books'));
+const BooksListsl = document.getElementById('Books-lists');
+
+// if local storage is empty then default books will store in local storage with key as books
+// which is store in books.json
+if (BooksArray === null || BooksArray.length === 0) {
+  localStorage.setItem('books', JSON.stringify(DefaultBooks));
+  BooksArray = DefaultBooks;
+}
+
+// Logic for removing data from local storage
+const remove = (RemovedBooks) => {
+  BooksArray = BooksArray.filter((book) => book.id !== RemovedBooks.id);
+  localStorage.setItem('books', JSON.stringify(BooksArray));
+  BooksListsl.removeChild(RemovedBooks);
+};
+
+// adding books into local storage
+const AddBookToStorage = (title, author, id) => {
+  BooksArray.push(
+    {
+      title,
+      author,
+      id,
+    },
+  );
+  localStorage.setItem('books', JSON.stringify(BooksArray));
+};
+
+// Showing books on UI interface
+const AddBookToDom = (title, author, index) => {
+  const newBookEl = document.createElement('li');
+  newBookEl.classList.add('book');
+  newBookEl.setAttribute('id', index);
+  newBookEl.innerHTML = `
+  <h2 class="book-title">${title}</h2>
+  <p class="book-author">${author}</p>
+  <button class="remove-btn">remove</button><hr>
+  `;
+  newBookEl.addEventListener('click', () => remove(newBookEl));
+  BooksListsl.appendChild(newBookEl);
+};
+
+// Creating list of books
+const createList = () => {
+  BooksArray.forEach((book) => {
+    AddBookToDom(book.title, book.author, book.id);
+  });
+};
+
+// Removing books from UI
+document.querySelectorAll('remove-btn').forEach((removeBtn) => {
+  removeBtn.addEventListener('click', remove(removeBtn));
+});
```

Comment:
- [ ] Your book removal only works on the page. It does not remove the book from the local storage as well. Kindly fix that.